bin/BuildPackages.sh

```diff
@@ -190,6 +191,19 @@ build_one_package() {
   (  # start subshell
   set -e
   cd "$CURDIR/$PKG"
+  local PKG_NAME=$($GAPROOT/gap -q -T <<GAPInput
+MakeReadWriteGlobal("SetPackageInfo");
+a:=1;;
+SetPackageInfo:= function(n)
+    a:=n;
+end;;
+Read("PackageInfo.g");
+Print(a.PackageName);
```

Comment:
```suggestion
Print(GAPInfo.PackageInfoCurrent.PackageName);
```

Then remove the five lines before `Read`, they are not necessary.